examples/aa-simple-dapp/src/hooks/useSessionKey.ts

```diff
@@ -0,0 +1,92 @@
+import { createLocalAccount } from "@/shared/util";
+import { IAccountLoupe, MSCA } from "@alchemy/aa-accounts";
+import { AlchemyProvider } from "@alchemy/aa-alchemy";
+import { LocalAccountSigner } from "@alchemy/aa-core";
+import { useCallback, useState } from "react";
+import { Address, Hex, PrivateKeyAccount } from "viem";
+import { useAsyncEffect } from "./useAsyncEffect";
+
+export const useSessionKey = ({
```

Comment:
I don't think we need this anymore right? since we can use the `usePlugin` + session key account factory to handle most of this stuff?